SEED-platform/seed
# 3140 bug/redirect 404 no sidebar

#### Any background context you want to provide?
When an unknown URL is entered a 404 error is initiated. SEED loads the sidebar, header, and 404 content. With the 404 page loaded the sidebar fails to lookup an active org and prompts the user to create a new one (similar to a user's first time visiting seed). Upon filling out a new organization, the user will be redirected to the home page where they will have a new organization attached to their profile.

#### What's this PR do?

When a 404 or 500 error is triggered the sidebar is no longer loaded, preventing the modal from opening.

#### How should this be manually tested?
To test locally in docker a few settings need to be modified. 
- /config/settings/docker_dev.py --> change `DEBUG` to `True` 
- /docker-compose.dev.yml --> add `--insecure` flag to the seed_web`command`:  `command:python3 manage.py runserver --insecure --settings=config.settings.docker_dev 0.0.0.0:80`

Go to any page that does not exist. You should be on a 404 page without a sidebar.

#### What are the relevant tickets?
#3140 

#### Screenshots

https://user-images.githubusercontent.com/58446472/155615245-aad71155-b013-41b6-abd0-1dd9c25cf634.mov
